CyclopsNuclearReactor/CyNukeReactorMono.cs

```diff
@@ -0,0 +1,427 @@
+﻿namespace CyclopsNuclearReactor
+{
+    using Common;
+    using MoreCyclopsUpgrades.Managers;
+    using ProtoBuf;
+    using System;
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    [ProtoContract]
+    internal partial class CyNukeReactorMono : HandTarget, IHandTarget, IProtoEventListener, IProtoTreeEventListener
+    {
+        internal static readonly string[] SlotNames = { "CyNuk1", "CyNuk2", "CyNuk3", "CyNuk4" };
```

Comment:
Forgot to add these to `slotMapping`.

Reply:
Because this is an entirely new `Equipment`, it may be necessary to also configure the UI placement of the slots.  
For reference, this is how it was done for the SlotExtender:  
[Initialize_uGUI](https://github.com/Senna7608/SennaSubnauticaPlugins/blob/master/SlotExtender/Initialize_uGUI.cs)
[uGUI_Equipment_Patcher](https://github.com/Senna7608/SennaSubnauticaPlugins/blob/master/SlotExtender/Patchers/uGUI_Equipment_Patcher.cs)

Reply:
Needs testing

Reply:
Converted to `ItemsContainer`